packages/vue/src/components/atoms/SfQuantitySelector/SfQuantitySelector.vue

```diff
@@ -99,6 +100,10 @@ export default {
   handleBlur(listeners) {
     return listeners.blur && listeners.blur();
   },
+  uniqueKey() {
+    const key = Math.random().toString(16).slice(2);
+    return "quantitySelector" + key;
```

Comment:
I would add hyphen between quantitySelector and key number for greater readability, wdyt?